From c06a5b0af0813e600da37e95589df31e06569f56 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Tue, 18 Nov 2014 13:54:50 -0500
Subject: [redhat] kabi: Adding symbol module_layout (kernel/module.c) MODULE SUPPORT

Message-id: <87139c184727c1b5d92b617172c15c884399e0b5.1416318767.git.poros@redhat.com>
Patchwork-id: 100160
O-Subject: [PATCH RHEL7.1 45/63] kabi: Adding symbol module_layout (kernel/module.c) MODULE SUPPORT
Bugzilla: 1070274
RH-Acked-by: Don Zickus <dzickus@redhat.com>

Bugzilla: 1070274
    https://bugzilla.redhat.com/show_bug.cgi?id=1070274

    Description:
    ============
    Used by kernel symbol versioning (CONFIG_MODVERSIONS). We do not have
    any influence on that.
    Symbol used in drbd

Symbol History
==============
 Request:
 ========
 [PATCH RHEL7.0 01/80] kabi: Adding symbol module_layout (kernel/module.c)
 Jiri Olsa <jolsa@redhat.com> (Tue,  4 Mar 2014 16:31:01 +0100)
 Description:
 ============
 Used by kernel symbol versioning (CONFIG_MODVERSIONS).

 Reviews:
 ========
 Re: [PATCH RHEL7.0 01/80] kabi: Adding symbol module_layout (kernel/module.c)
 Prarit Bhargava <prarit@redhat.com> (Wed, 05 Mar 2014 07:52:16 -0500)
 Acked-by: Prarit Bhargava <prarit@redhat.com>
 P.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/module_layout
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/module_layout
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/module_layout

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/module_layout b/redhat/kabi/kabi-module/kabi_ppc64/module_layout
new file mode 100644
index 0000000..30dc376
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/module_layout
@@ -0,0 +1,2 @@
+#1-
+0x8d1fd821 module_layout vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/module_layout b/redhat/kabi/kabi-module/kabi_s390x/module_layout
new file mode 100644
index 0000000..e755ed4
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/module_layout
@@ -0,0 +1,2 @@
+#1-
+0x033edbda module_layout vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/module_layout b/redhat/kabi/kabi-module/kabi_x86_64/module_layout
new file mode 100644
index 0000000..c6ea644
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/module_layout
@@ -0,0 +1,2 @@
+#1-
+0x28950ef1 module_layout vmlinux EXPORT_SYMBOL
-- 
1.7.1